src/Cake.Common/Tools/MSBuild/MSBuildTarget.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Cake.Common.Tools.MSBuild
+{
+    /// <summary>
+    /// Represents an MSBuild exe target.
+    /// </summary>
+    public enum MSBuildTarget
```

Comment:
Should be renamed to `MSBuildPlatform` to avoid confusion with [MSBuild targets](https://msdn.microsoft.com/en-us/library/ms171462.aspx). Will also follow Team Builds approach to this.